### DISCUSSION TITLE
Create personalized report

### DISCUSSION THREAD

**Discussion by liekederouw** [Samedis-care/samedis-care-community#41] [Ideas]
Under "actions" it is possible to select medical products book. A lot of inventory information appears on the sheet, however information in the "notes" section (for example) does not appear here. AZP would like to have all the information in one document file. Would this be a possibility in the future?

**Comment by YvesR** [comment]
Can you elaborate in detail what fields you are missing so we can check this out?

**Reply by liekederouw** [reply]
we would like all the fields we fill in per inventory on one sheet.

**Comment by BioMedGPH** [comment]
maybe it is an idea to offer a settings page somewhere and allow to set what info will go on the medical products book?
